packages/design-toolkit/src/components/flashcard/index.tsx

```diff
@@ -0,0 +1,200 @@
+ */
+
+/**
+ * Example usage.
+ *
+ * <Flashcard>
+ *  <FlashcardHero>
+ *    <FlashcardIdentifier>
+ *      <FlashcardHeader>
+ *        {header}
+ *      </FlashcardHeader>
+ *      <FlashcardSubheader>
+ *        {subHeader}
+ *      </FlashcardSubheader>
+ *    </FlashcardIdentifier>
+ *  </FlashcardHero>
+ *  <FlashcardSecondary>
+ *    <FlashcardSecondaryData>
+ *      {secondaryData}
+ *    </FlashcardSecondaryData>
+ *    <FlashcardDetailsContainer details={details} />
+ *  </FlashcardSecondary>
+ * </Flashcard>
+ */
+'use client';
+import 'client-only';
+import { createContext, useContext } from 'react';
+import { Skeleton } from '../skeleton';
+import { FlashcardStyles } from './styles';
+import type {
+  FlashcardComponentProps,
+  FlashcardDetailContainerProps,
+  FlashcardProps,
+} from './types';
+
+const {
+  container,
+  hero,
+  identifier,
+  header,
+  subHeader,
+  secondaryContainer,
+  secondaryData,
+  detailsContainer,
+  detailsItem,
+  detailsLabel,
+  detailsValue,
+  skeleton,
+} = FlashcardStyles();
+
+export const FlashcardContext = createContext<FlashcardProps>({
+  isLoading: false,
+});
+
+export function Flashcard(props: FlashcardProps) {
+  const { isLoading, children, className, ...rest } = props;
+
+  return (
+    <FlashcardContext.Provider value={{ isLoading }}>
+      <div {...rest} className={container({ className })}>
+        {children}
+      </div>
+    </FlashcardContext.Provider>
+  );
+}
+Flashcard.displayName = 'Flashcard';
+
+export function FlashcardHero(props: FlashcardComponentProps) {
+  const { children, className, ...rest } = props;
+  const { isLoading } = useContext(FlashcardContext);
+
+  if (isLoading) {
+    return (
+      <div {...rest} className={hero({ className })}>
+        <Skeleton className={skeleton()} />
+        <Skeleton className={skeleton({ className: 'max-w-1/2' })} />
+      </div>
+    );
+  }
+
+  return (
+    <div {...rest} className={hero()}>
+      {children}
+    </div>
+  );
+}
+FlashcardHero.displayName = 'FlashcardHero';
+
+export function FlashcardIdentifier(props: FlashcardComponentProps) {
+  const { children, className, ...rest } = props;
+
+  return (
+    <div {...rest} className={identifier({ className })}>
+      {children}
+    </div>
+  );
+}
+FlashcardIdentifier.displayName = 'FlashcardIdentifier';
+
+export function FlashcardHeader(props: FlashcardComponentProps) {
+  const { children, className, ...rest } = props;
+
+  return (
+    <div {...rest} className={header({ className })}>
+      {children}
+    </div>
+  );
+}
+FlashcardHeader.displayName = 'FlashcardHeader';
+
+export function FlashcardSubheader(props: FlashcardComponentProps) {
+  const { children, className, ...rest } = props;
+
+  return (
+    <div {...rest} className={subHeader({ className })}>
+      {children}
+    </div>
+  );
+}
+FlashcardSubheader.displayName = 'FlashcardSubheader';
+
+export function FlashcardSecondary(props: FlashcardComponentProps) {
```

Comment:
I think you can get rid of this component entirely. Since it seem to be solely responsible for a bit of layout, that can be handled by the Flashcard itself. I would potentially reach for a 5 column grid. The 1st, 3rd, and 5th columns being gutters, 2nd and 4th being named. Hero spans all columns, secondary data spans 2-4, and the details list is on the two named columns label and value

Reply:
Scratch that, that's too complicated. Just apply the padding to the Flashcard and use negative margin on the hero and the details list